jbot/src/main/java/me/ramswaroop/jbot/core/slack/Bot.java

```diff
@@ -102,7 +102,7 @@ public void afterConnectionEstablished(WebSocketSession session) {
      * @see WebSocketHandler#afterConnectionClosed
      */
     public void afterConnectionClosed(WebSocketSession session, CloseStatus status) {
-        logger.debug("WebSocket closed: {}, Close Status: {}", session, status.toString());
```

Comment:
I guess this change was already present in your previous PR and is now merged to master. If possible can you please exclude this change from this PR?